### ISSUE TITLE
About training details

### ISSUE THREAD

**Issue by SeonjeongHwang** [LHRYANG/Generalization_of_FT-LLM#2]
Hello.
Thank you for your interesting research.

Can you share the hardware environment for fine-tuning the Llamma2-7b model?  
I'm curious about which GPUs and how many were used. 
Also, the paper says that the learning rate was set to 0.002, but it is written as 2e-5 in run_train.sh.
What value should I use?

I would appreciate your reply.

**Comment by LHRYANG** [comment]
Sorry for late reply. For GPU, I use 2 A6000 to train the model. For the learning rate, I use 2e-5.
